### ISSUE TITLE
Using Voxel-RCNN for detector

### ISSUE THREAD

**Issue by minju-Kang** [azhuantou/HSSDA#17]
Did you use 80 training epoch for voxel-rcnn as for pv-rcnn?
I run your code with voxel rcnn detector, voxel-rcnn seems converge slowly than pv-rcnn.
Here are figures for training loss over epochs. (Above: voxel-rcnn with 160 epochs, Below: pv-rcnn with 80 epochs)
<img width="206" alt="voxel_rcnn_ssl" src="https://github.com/azhuantou/HSSDA/assets/54900895/7d89e4f0-9d0f-49c6-b6a9-06ed6a9c4ffe">
<img width="213" alt="pv_rcnn_ssl" src="https://github.com/azhuantou/HSSDA/assets/54900895/3cd6db2d-1e24-44d7-b23c-f1daef63446f">
Could you share the training loss graph?
By the way, thank you for sharing your work!

**Comment by azhuantou** [comment]
Hi, thanks for your interests！
The above figure and below figure are the training loss change process  for PV-RCNN and Voxel-RCNN, respectively.
![T2$ZILFJV$}EOH@`NO5WBB0](https://github.com/azhuantou/HSSDA/assets/33474693/6fb302bc-5b4e-472e-a067-a501269ca2a5)
![1X9W$7)42%G8VHK92RXRO3E](https://github.com/azhuantou/HSSDA/assets/33474693/365e75a8-c220-447d-8dcb-80441b849779)
